backend/README.md

```diff
@@ -67,8 +67,8 @@ Var | Meaning
 ## Docker
 
 ```bash
-docker build . -t codeforcharlottesville/housinghubapi
-docker run -p 5000:5000 codeforcharlottesville/housinghubapi
+docker build . -t codeforcharlottesville/housinghubapi:alpine-3.6-slim
+docker run -p 5000:5000 codeforcharlottesville/housinghubapi:alpine-3.6-slim
 ```
```

Comment:
we can probably keep the default `latest` image here. I don't think we will need a specification of `alpine-3.6-slim`

Reply:
yes, you are right thanks! it was more for differencing the 3 sizes